src/log-tlslog.c

```diff
@@ -291,8 +297,11 @@ static int LogTlsLogger(ThreadVars *tv, void *thread_data, const Packet *p,
         return 0;
     }
 
-    if (ssl_state->server_connp.cert0_issuerdn == NULL ||
-            ssl_state->server_connp.cert0_subject == NULL) {
+    if ((ssl_state->server_connp.cert0_issuerdn == NULL ||
+            ssl_state->server_connp.cert0_subject == NULL) &&
+        ((ssl_state->flags & SSL_AL_FLAG_SESSION_RESUMED) == 0 ||
+         (hlog->flags & LOG_TLS_SESSION_RESUMPTION) == 0)) {
```

Comment:
The indentation for the various parts of this "if" is not aligned with each other.